common/src/main/java/com/wynntils/core/functions/expressions/FunctionExpression.java

```diff
@@ -93,21 +99,12 @@ public static ErrorOr<Optional<Expression>> tryParse(String rawExpression) {
 
         // Handle argument parsing
 
-        FunctionArguments.Builder argumentsBuilder = function.getArguments();
-
-        if (matcher.groupCount() < 3) {
-            return ErrorOr.of(Optional.of(new FunctionExpression(
-                    rawExpression, function, argumentsBuilder.buildWithDefaults(), isFormatted, decimals)));
-        }
+        FunctionArguments.Builder argumentsBuilder = function.getArgumentsBuilder();
```

Comment:
Just checking here. This is done at parse time, right? Which for info boxes is done only on config update time, not at render time..? (Sorry, I'm being a bit lazy and try to skip having to delve into the code to check this...)

Reply:
Yes, updated on parse time, configurable for info boxes

Reply:
Oh. I just realized. Well, it is done on calculation time, which is "render" time (every 5 ticks). Yes, I know that is not so optimal. I would wait with optimizing though. If we profile this, it might not even show up (or be significant).